front/components/TrixWrapper.vue

```diff
@@ -191,6 +212,13 @@ export default {
             href: source,
             key
           })
+
+          // Save uploaded file for CTRL-Z case
+          const existingIdx = this.uploadedFiles.findIndex(
+            a => a.id === event.attachment.id
+          )
+          if (existingIdx) this.uploadedFiles.splice(existingIdx, 1)
```

Comment:
please use {} for if, even if its one line, its easier to understand and when changing code less margin for errors

Reply:
@aldarund Lets then add a rule for this https://eslint.org/docs/rules/curly

Reply:
Sounds good, please add